interchain_test.go

```diff
@@ -206,7 +206,7 @@ func TestInterchain_CreateUser(t *testing.T) {
 
 		user := ibctest.GetAndFundTestUserWithMnemonic(t, ctx, keyName, mnemonic, 10000, gaia0)
 		require.NoError(t, test.WaitForBlocks(ctx, 2, gaia0))
-		require.NotEmpty(t, user.Address)
+		require.NotEqual(t, "", user.Address)
```

Comment:
nit:

A more suitable matcher here is `require.NotEmpty`

Reply:
Or `require.NotZero` which in Go, refers to [zero values](https://go.dev/tour/basics/12).

Reply:
resolved with 
https://github.com/strangelove-ventures/ibctest/pull/255